### PR TITLE
Add SRT caller support

### PR DESCRIPTION
**Pull request by qwu16**

### CODE REVIEW

File: scripts/release/install_ffmpeg.sh [hunk 1]
```diff
@@ -57,16 +57,40 @@ install_build_deps() {
   then
     echo -e "\x1b[32mInstalling dependent components and libraries via yum...\x1b[0m"
     ${SUDO} yum install pkg-config make gcc gcc-c++ nasm yasm freetype-devel -y
+    if [ "$ENABLE_SRT" = "true" ]; then
```
Comment (daijh) [inline scripts/release/install_ffmpeg.sh:60]: Seem we do not need it.

File: scripts/release/install_ffmpeg.sh [hunk 2]
```diff
@@ -57,16 +57,40 @@ install_build_deps() {
...
+	${SUDO} yum install tcl openssl-devel cmake automake -y
+    fi
   elif [[ "$OS" =~ .*ubuntu.* ]]
   then
     echo -e "\x1b[32mInstalling dependent components and libraries via apt-get...\x1b[0m"
     ${SUDO} apt-get update
     ${SUDO} apt-get install pkg-config make gcc g++ nasm yasm libfreetype6-dev -y
+    if [ "$ENABLE_SRT" = "true" ]; then
```
Comment (daijh) [inline scripts/release/install_ffmpeg.sh:68]: Not needed too.

File: scripts/release/install_ffmpeg.sh [hunk 3]
```diff
@@ -88,8 +112,13 @@ install_ffmpeg(){
   rm -fr ${DIR}
   tar xf ${SRC}
   pushd ${DIR} >/dev/null
-  CFLAGS=-fPIC ./configure --prefix=${PREFIX_DIR} --enable-shared \
-    --disable-static --disable-libvpx --disable-vaapi --enable-libfreetype
+  if [ "$ENABLE_SRT" = "true" ]; then
```
Comment (daijh) [inline scripts/release/install_ffmpeg.sh:115]: maybe not need

File: scripts/release/install_ffmpeg.sh (hunk 1 above)
Comment (starwarfan) [reply scripts/release/install_ffmpeg.sh:69]: This is not only used by streaming-agent

**Review by starwarfan — APPROVED** [review]

**Pull request merged**